app/models/order.rb

```diff
@@ -6,7 +6,7 @@ class Order < ApplicationRecord
   has_many :items, through: :item_orders
   belongs_to :user
 
-  # enum status: %w(pending packaged shipped cancelled)
+  enum status: %w(pending packaged shipped cancelled)
```

Comment:
This is the exact issue that I was looking at in 32! This will fix my challenge.

So will these then have values of pending - 0, packaged - 1 and so on?